app/javascript/controllers/flash_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class AdminFlashController extends Controller {
+  static targets = ['close'];
+
+  setCookie() {
```

Comment:
What do you think about renaming this function to `disable()` or `hide()`? we do set a cookie in it, but the cookie is used to prevent the admin flash from displaying again. One of those names would better describe the overall behaviour imo.